packer.mk

```diff
@@ -106,3 +106,13 @@ build-nbcparser-all:
 build-nbcparser-binary:
 	@echo "Building nbcparser binary"
 	@bash -c "pushd nbcparser && CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) go build -o bin/nbcparser-$(ARCH) main.go && popd"
+
+build-lister-binary:
+ifeq (${ARCHITECTURE},X86_64)
+	@echo "Building lister binary for amd64"
+	@bash -c "pushd vhdbuilder/lister && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o bin/lister main.go && popd"
```

Comment:
could we use a variable for the GOARCH and only have one `go build` line ?

```
GOARCH=amd64
ifeq (${ARCHITECTURE},ARM64)
    GOARCH=arm64
endif
@echo "Building lister binary for ${GOARCH}"
@bash -c "pushd vhdbuilder/lister && CGO_ENABLED=0 GOOS=linux GOARCH=${GOARCH} go build -o bin/lister main.go && popd"
```

Reply:
I tried that exact approach and I was having trouble getting it working, I can revisit though

Reply:
refactored